test.disabled/ejabberd_tests/tests/login_SUITE.erl

```diff
@@ -404,7 +404,12 @@ log_one_scram(Config) ->
 
 
 log_non_existent_plain(Config) ->
-    {auth_failed, _, _} = log_non_existent(Config).
+    {auth_failed, _, Xmlel} = log_non_existent(Config),
+    {xmlel, <<"failure">>, _Attrs, Data} = Xmlel,
```

Comment:
Although matching on `#xmlel{}` as a tuple can still be seen sometimes in old code, in the new one we always use record syntax. So here we should have `#xmlel{ name = <<"failure">> } = `